### PR TITLE
DF-261:Add django debug toolbar and Setup Local development for DEBUG=False

### PR DESCRIPTION
**Pull request by TNA-Allan**
Added django debug toolbar to display various debug information in debug mode
Setup local development for Debug=False

### CODE REVIEW

File: docs/developer-guide/local-development-gotchas.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
...
+DEBUG = False
+STATICFILES_STORAGE = "django.contrib.staticfiles.storage.StaticFilesStorage"
+```
+
+https://docs.djangoproject.com/en/3.2/ref/contrib/staticfiles/#cmdoption-runserver-insecure
+Run
+```console
+python manage.py runserver 0.0.0.0:8000 --insecure
+```
+
```
Comment (ababic) [inline docs/developer-guide/local-development-gotchas.md:21]: I was sort of expecting these to be listed as sub-sections below the introductory content. Also, not sure that this explains things that well:

> The toolbar is enabled for these variable settings which is default in dev.

Maybe something like:

```
## How do I disable Django's 'debug' mode?

Sometimes you need to disable Django's DEBUG mode in order to test certain behaviours. To do this, you have two options:

- Add `DEBUG = False` to `conf/settings/local.py` 
- Add `DEBUG=False` to your `.env` file

## How do I disable Django Debug Toolbar?

If you don't need Django's debugging tools, setting `DEBUG=False` as detailed above will disable Django Debug Toolbar also.

If you still need Django's debugging tools, but don't want to use Django Debug Toolbar for any reason, you have two options for disabling it:

- Add `DEBUG_TOOLBAR_ENABLED = False` to `conf/settings/local.py` 
- Add `DEBUG_TOOLBAR_ENABLED=False` to your `.env` file
```

Comment (TNA-Allan) [reply docs/developer-guide/local-development-gotchas.md]: Done

File: docs/developer-guide/local-development-gotchas.md [hunk 2]
```diff
@@ -1,7 +1,30 @@
-# Local development gotchas
+## How do I disable Django's 'debug' mode?
```
Comment (ababic) [inline docs/developer-guide/local-development-gotchas.md:1]: I think the main heading should remain as "Local development gotchas"

File: docs/developer-guide/local-development-gotchas.md [hunk 3]
```diff
@@ -1,7 +1,30 @@
...
 
-This page lists common local development issues and how do resolve them.
+Sometimes you need to disable Django's DEBUG mode in order to test certain behaviours. To do this, you have two options:
```
Comment (ababic) [inline docs/developer-guide/local-development-gotchas.md:3]: I think the original introductory text here is also useful

File: docs/developer-guide/local-development-gotchas.md [hunk 4]
```diff
@@ -1,7 +1,30 @@
...
 
-## Brief description of problem
+1. Add `DEBUG = False` to `config/settings/local.py` 
+2. Add `DEBUG=False` to your `.env` file
```
Comment (ababic) [inline docs/developer-guide/local-development-gotchas.md:6]: Try adding `[TOC]` here instead for functional jump/skip llnks

File: docs/developer-guide/local-development-gotchas.md [hunk 5]
```diff
@@ -1,7 +1,30 @@
...
+## How do I disable Django Debug Toolbar?
+
+If you don't need Django's debugging tools, setting `DEBUG=False` as detailed above will disable Django Debug Toolbar also.
+
+If you still need Django's debugging tools, but don't want to use Django Debug Toolbar for any reason, you have two options for disabling it:
+
+1. Add `DEBUG_TOOLBAR_ENABLED = False` to `config/settings/local.py` 
+2. Add `DEBUG_TOOLBAR_ENABLED=False` to your `.env` file
+
+## How do I run with 'DEBUG=False' in dev ?
```
Comment (ababic) [inline docs/developer-guide/local-development-gotchas.md:17]: Would prefer something non-techy, like:
```
How do I disable Django's 'Debug mode' locally?
```

File: docs/developer-guide/local-development-gotchas.md (hunk 3 above)
Comment (TNA-Allan) [reply docs/developer-guide/local-development-gotchas.md]: Done, kept original text

File: docs/developer-guide/local-development-gotchas.md (hunk 5 above)
Comment (TNA-Allan) [reply docs/developer-guide/local-development-gotchas.md]: Done. Thank you Andy for these comments docs

File: docs/developer-guide/local-development-gotchas.md [hunk 6]
```diff
@@ -1,7 +1,32 @@
 # Local development gotchas
-
```
Comment (ababic) [inline docs/developer-guide/local-development-gotchas.md:2]: Could we keep the blank line between the heading and paragraph?

File: docs/developer-guide/local-development-gotchas.md [hunk 7]
```diff
@@ -1,7 +1,32 @@
...
 This page lists common local development issues and how do resolve them.
 
-## Brief description of problem
 
-Solution description goes here.
+Sometimes you need to disable Django's DEBUG mode in order to test certain behaviours. To do this, you have two options:
+
+1. Add `DEBUG = False` to `config/settings/local.py` 
+2. Add `DEBUG=False` to your `.env` file
```
Comment (ababic) [inline docs/developer-guide/local-development-gotchas.md:8]: Shouldn't this content be under one of the headings below?

Comment (TNA-Allan) [reply docs/developer-guide/local-development-gotchas.md]: Done

File: docs/developer-guide/local-development-gotchas.md (hunk 6 above)
Comment (TNA-Allan) [reply docs/developer-guide/local-development-gotchas.md:2]: Done

File: docs/developer-guide/local-development-gotchas.md (hunk 4 above)
Comment (TNA-Allan) [reply docs/developer-guide/local-development-gotchas.md]: We decided not to keep it at this time.

**Review by ababic — APPROVED** [review]
Thanks @TNA-Allan! LGTM!

**Pull request merged**